app/controllers/partials/journey_hinting_partial_controller.rb

```diff
@@ -44,6 +44,11 @@ def last_rp
     last_status_value.nil? ? nil : last_status_value.fetch("RP", nil)
   end
 
+  def last_journey_type
```

Comment:
```suggestion
  def last_verify_journey_type
```

There already is `user_session_type` method in `UserSessionPartialController` to distinguish between Verify and eIDAS journeys. The journey type here is the "Verify journey type" from the Verify module in `JourneyType`. Maybe better naming is needed to minimise confusion.

https://github.com/alphagov/verify-frontend/blob/6571fcbdf14d204a6bae4f589570056bab955ce7/app/controllers/partials/user_session_partial_controller.rb#L33-L40

Reply:
Makes sense, will update.